model/config.go

```diff
@@ -210,20 +215,11 @@ type IstioConfigStore interface {
 }
 
 const (
-	// RouteRule defines the type for the route rule configuration
-	RouteRule = "route-rule"
-	// RouteRuleProto message name
-	RouteRuleProto = "istio.proxy.v1.config.RouteRule"
-
-	// IngressRule type
-	IngressRule = "ingress-rule"
-	// IngressRuleProto message name
-	IngressRuleProto = "istio.proxy.v1.config.IngressRule"
+	// IstioAPIGroup defines API group name for Istio configuration resources
+	IstioAPIGroup = "config.istio.io"
 
-	// DestinationPolicy defines the type for the destination policy configuration
-	DestinationPolicy = "destination-policy"
-	// DestinationPolicyProto message name
-	DestinationPolicyProto = "istio.proxy.v1.config.DestinationPolicy"
+	// IstioAPIVersion defines API group version
+	IstioAPIVersion = "v1alpha1"
```

Comment:
I remember seeing bug reports about using two versions for same CRD group. I changed it to v1alpha2 with the goal of coordinating the change with @mandarjog